### DISCUSSION TITLE
Is it possible to enable landscape table pages in rinohtype or rst?

### DISCUSSION THREAD

**Discussion by th0mr** [brechtm/rinohtype#391] [Q&A] [answered]
I am attempting to build some documents that have a few large tables. 

It would be ideal if there existed some way to specify that all / some tables are put on a landscape page. Is this possible at the moment with rinohtype?

Alternatively, I was wondering how to utilise the new support for sideways figures and tables that was mentioned in the rinohtype 0.5.4 release notes. It mentions the Float style attribute, but when I attempt to incorporate float into my page-break-table styling as shown below, I get "TypeError: float is not a supported attribute for TableStyle". Is there a way to ensure the Table class incorperates the float option, is there something I'm missing perhaps?

```
[page-break-table : Table(has_class="page-break")]
base=table
page_break=any
float=sideways
```

 I'm still new to using rinohtype, any help is greatly appreciated.

**Comment by brechtm** [comment, marked as answer]
Sorry, this topic got lost in my chaotic inbox!

> It would be ideal if there existed some way to specify that all / some tables are put on a landscape page. Is this possible at the moment with rinohtype?

This is exactly what the sideways figures/tables in 0.5.4 offer! If you want all tables to be placed on a landscape page, you can set `float = sideways` for the default _table_ style definition. If you want only some tables to be sideways, your _page-break-table_ definition should work. Of course, you then need to set the "page break" class on these tables in reStructuredText.

> Alternatively, I was wondering how to utilise the new support for sideways figures and tables that was mentioned in the rinohtype 0.5.4 release notes. It mentions the Float style attribute, but when I attempt to incorporate float into my page-break-table styling as shown below, I get "TypeError: float is not a supported attribute for TableStyle". Is there a way to ensure the Table class incorperates the float option, is there something I'm missing perhaps?

Only tables with a caption support floating. While I think it's good practice to provide a caption for a table that is rendered out-of-line, it should probably be possible to float caption-less tables as well. For now, try adding a caption to your table.

See the `table_sideways.*` files in https://github.com/brechtm/rinohtype/tree/master/tests_regression/rst for an example.
